### ISSUE TITLE
Fix test_wsgi.StartResponseTests on PyPy

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8629]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8629|
|Type|enhancement|
|Created|2016-07-19 20:07:47Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8629 8629
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__None None
time__1468958867124979 1468958867124979
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/357

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 620a3bfeb7034b6aa7cf36fc46d02c90d48afa60
```
#!CommitTicketReference repository="" revision="620a3bfeb7034b6aa7cf36fc46d02c90d48afa60"
Merge pull request #362 from twisted/8629-scop-pypywsgi

Author: scop
Reviewer: rodrigc
Fixes: #8629

Fix test_wsgi.StartResponseTests on PyPy
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
